test/browser/router.js

```diff
@@ -38,7 +38,7 @@ describe(`Router`, function () {
     this.routerApp = document.createElement(`router-app`);
     document.body.appendChild(this.routerApp);
 
-    await nextAnimationFrame();
+    return nextAnimationFrame();
```

Comment:
weird that this makes a difference? also if you do that then the function shouldn't be marked `async` anymore

Reply:
oh this should be a mistake, reverted